ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/TerminateCluster.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.rest.util;
+
+import io.confluent.ksql.parser.tree.Statement;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+public class TerminateCluster extends Statement {
```

Comment:
Consider adding an enum here with valid values for `SOURCES_LIST_TYPE_PARAM_NAME`. You can then use these to validate the values. (Maybe via a static method on this class).

Reply:
Using enum now as you suggested.